dotcom-rendering/src/components/LoopVideo.importable.tsx

```diff
@@ -57,26 +72,54 @@ export const LoopVideo = ({
 	});
 
 	/**
+	 * Setup.
+	 *
 	 * Register the users motion preferences.
+	 * Creates an event listener to ensure we don't play audio from multiple loops
 	 */
 	useEffect(() => {
 		const userPrefersReducedMotion = window.matchMedia(
 			'(prefers-reduced-motion: reduce)',
 		).matches;
 		setPrefersReducedMotion(userPrefersReducedMotion);
-	}, []);
+
+		/**
+		 * Pause the current video when another video is played
+		 * Triggered by the CustomEvent sent by each player on play
+		 */
+		const handleCustomPlayAudioEvent = (
+			event: CustomEventInit<CustomPlayEventDetail>,
+		) => {
+			if (event.detail) {
+				const playedVideoId = event.detail.uniqueId;
+				const thisVideoId = uniqueId;
+
+				if (playedVideoId !== thisVideoId) {
+					setIsMuted(true);
+				}
+			}
+		};
+
+		document.addEventListener(
+			customPlayAudioEventName,
+			handleCustomPlayAudioEvent,
+		);
```

Comment:
I think we should be removing this as well when the component is unmounted to clean up eg 

`return () => document.removeEventListener(customPlayAudioEventName, handleCustomPlayAudioEvent)`